web/app/themes/ppj/src/sass/components/header.sass

```diff
@@ -42,59 +55,105 @@
     top: 0
     width: 100%
 
-  &__site-header
-    background: linear-gradient(to bottom, $color-transparency-dark-start, $color-transparent)
+  &__nav
     box-sizing: border-box
     z-index: 1
 
-    &-content
-      @extend .l-full
-      display: flex
-      justify-content: space-between
-      margin-bottom: 0
-      margin-top: 0
-      opacity: 1
-      padding: $common-spacing
+  &__site-wide-nav
```

Comment:
`.header__site-wide-nav`

Should the site wide nav be a BEM element of header? This seems unnecessarily nested and restrictive to me, plus it bloats out the responsibility of the `header` component.

`header.sass` currently sits at ~500 lines, which feels a little on the heavy side for one component.

I think `site-wide-nav` is complex enough (both in terms of styling & JS functionality) to be a component of its own. The same goes for the leg nav.

Reply:
done